### PR TITLE
Update unity editor

### PR DESCRIPTION
**Pull request by cargy**
Just updating to the latest LTS Unity Editor 2022.3

### CODE REVIEW

File: DeckSwipe/ProjectSettings/UnityConnectSettings.asset [hunk 1]
```diff
@@ -4,11 +4,12 @@
 UnityConnectSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 1
-  m_Enabled: 0
+  m_Enabled: 1
```
Comment (outfrost) [inline DeckSwipe/ProjectSettings/UnityConnectSettings.asset:7]: Does this enable Unity online services, where previously they'd be disabled? I feel like that doesn't need to be the default.

File: DeckSwipe/Assets/Resources/BillingMode.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"androidStore":"GooglePlay"}
```
Comment (outfrost) [inline DeckSwipe/Assets/Resources/BillingMode.json:1]: Is this created automatically when opening with the newer editor?

File: DeckSwipe/ProjectSettings/UnityConnectSettings.asset (hunk 1 above)
Comment (cargy) [reply DeckSwipe/ProjectSettings/UnityConnectSettings.asset:7]: Makes sense, but every time I build the project it changes back to `1`. Tried disabling the *CLOUD* option to *NOT CONNECTED* in Unity Hub for the project but it doesn't seem to have any impact on the `m_Enabled` property (turns back on after build).

File: DeckSwipe/Assets/Resources/BillingMode.json (hunk 2 above)
Comment (cargy) [reply DeckSwipe/Assets/Resources/BillingMode.json:1]: That's right. Even if you don't the Android module installed for the editor version.

File: DeckSwipe/ProjectSettings/UnityConnectSettings.asset (hunk 1 above)
Comment (outfrost) [reply DeckSwipe/ProjectSettings/UnityConnectSettings.asset:7]: Fair enough

**Review by outfrost — APPROVED** [review]

**Comment by outfrost** [comment]
Thanks!

**Pull request merged**